src/pages/getting-started/app-guides/integrations/index.mdx

```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
   
 </div>
 
+# Netlify and Vercel
+
+Connecting your app with Netlify or Vercel deployment service makes your development cycles faster, collaborations are more streamlined, and many things that must be done manually are automated. In a few seconds, the seamless deployment management allows you to get the URL of our application and configure all the building blocks your app needs easier and quicker.
+
+## Getting started
+
+Having the [GitHub repository linked to your app](#github) in the <Placeholder id="DEVELOPER_CONSOLE_URL" />, select the app you want to connect and go to the **Deployment** tab. You need to decide which deployment service you choose. Both flows are just as simple, consisting of only a few steps.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Deployment block" src="/images/getting-started/app-guides/integrations/livechat-deployment-block-dev-console.png" width="1406px" height="750px"/>
```

Comment:
```suggestion
  <img alt="LiveChat Dev Console Deployment block" src="/images/getting-started/app-guides/integrations/livechat-deployment-block-dev-console.png" width="1558px" height="767px"/>
```

Reply:
the images were slighty zwężone - this should fix them 🙆

Reply:
I and the images is some neverending story😅